jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/JSON.java

```diff
@@ -261,6 +261,13 @@ public enum Feature
          */
         USE_IS_GETTERS(true, true),
 
+        /**
+         * Feature that provides support for Groovy & JDK14 records, by allowing
+         * reading of "non-get-getters" in a class, (like for a field named <code>amount</code>
+         * the getter would be <code>amount()</code>)
+         * */
+        USE_FIELD_NAME_GETTERS(false,true),
```

Comment:
I am trying to think of naming... this is accurate and might work. But wondering if we could think of even better name. `USE_` is right prefix. I wonder if... something like `USE_PLAIN_GETTERS` would better capture the.

Or, better yet, `USE_FIELD_MATCHING_GETTERS` ?